tests/sympc/tensor/mpc_tensor_test.py

```diff
@@ -363,3 +363,20 @@ def test_share_get_method_session(get_clients) -> None:
     res = mpc_tensor * mpc_tensor
 
     assert all(res.get() == expected_res)
+
+
+def test_pow(get_clients) -> None:
+    clients = get_clients(2)
+    session = Session(parties=clients)
+    SessionManager.setup_mpc(session)
+
+    x_secret = torch.Tensor([5.0])
```

Comment:
MAYBE: Also add a negative value and parametrize this test to raise the tensor to an odd and even power

Reply:
done